frontend/src/scenes/surveys/QuestionBranchingInput.tsx

```diff
@@ -0,0 +1,68 @@
+import './EditSurvey.scss'
+
+import { LemonSelect } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { LemonField } from 'lib/lemon-ui/LemonField'
+
+import { MultipleSurveyQuestion, RatingSurveyQuestion, SurveyQuestionBranchingType } from '~/types'
+
+import { surveyLogic } from './surveyLogic'
+
+export function QuestionBranchingInput({
+    questionIndex,
+    question,
+}: {
+    questionIndex: number
+    question: RatingSurveyQuestion | MultipleSurveyQuestion
+}): JSX.Element {
+    const { survey, getBranchingDropdownValue } = useValues(surveyLogic)
+    const { setQuestionBranching } = useActions(surveyLogic)
+
+    const availableNextQuestions = survey.questions
```

Comment:
for a follow up: Also prevent infinite loops from being saved 😹

Reply:
Example: Q1 goes to Q3 which goes to Q1

Reply:
Hmm good point 👍